go/cbrotli.go

```diff
@@ -0,0 +1,264 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Distributed under MIT license.
+// See file LICENSE for detail or copy at https://opensource.org/licenses/MIT
+
+// Package cbrotli compresses and decompresses data with C-Brotli library.
+package cbrotli
```

Comment:
The convention in the Go community is to name the parent directory the same as package name.

https://golang.org/doc/effective_go.html#package-names says:
> Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64.

Thus, creating a directory `go/cbrotli` and placing this file in `go/cbrotli/cbrotli.go`. It is a redundant when there is only 1 package, but as a convention, it scales better.

Bazel allows for both the old-style and current representation (because Bazel supports the package layout that preceded the design of today's conventional layout), but this is not the norm for most Go users.